File: Rahul Dhanola/DAY - 2/question2.py

n=int(input("enter the integer: "))

if(n>0):
    year=n//365
    
    n%=365
    
    months=n//31
    
    n%=31
    
    
    
    print("the number of:\nyears =",year,"\nmonths =",months,"\ndays =",n)
    
else:
    print("Please enter the corect date!!! ")
    

File: Rahul Dhanola/DAY - 2/question2.c

#include <stdio.h>


int main(){
    
    int days,year,months;

    printf("\nenter the integer: ");
    
    scanf("%d",&days);

    if(days>0){
        
        year=days/365;
    
        days%=365;
    
        months=days/31;
    
        days%=31;
    
        printf("\nthe number of:\nyears = %d\nmonths = %d\ndays = %d\n",year,months,days);
        
    }

    else{
        
        
        printf("\nPlease enter the corect date!!! ");
        
    }
        
    
    return 0;
    
}